Cargo.toml

```diff
@@ -63,6 +63,8 @@ serde_yaml = "0.8"
 smash-bgm-property = "1.2.0"
 # For inputs
 ninput = { git = "https://github.com/blu-dev/ninput" }
+# For reading in NRR hashes
+binrw = "0.11.2"
```

Comment:
Genuinely don't think you need a heavy dependency like binrw to read such a simple file? Can you really need not parse if with regular Rust methods?

Reply:
The latest commit addresses this, however, I have yet to test the change on console.